executor/src/executor_test.go

```diff
@@ -3,10 +3,26 @@ package executor
 import (
 	"strings"
 	"testing"
+	"github.com/rs/zerolog"
+	"os"
 )
 
+func initLogger() (zerolog.Logger){
```

Comment:
Same thing as the compiler logs, I think this should be silent.

Reply:
Removed